feat: rules refactoring / add separate arazzo rule-set (#1800)
Redocly/redocly-cli: docs/rules/arazzo/struct.md

---
slug: /docs/cli/rules/arazzo/struct
---

# struct

Ensures that your API document conforms to the [OpenAPI specification](https://spec.openapis.org/arazzo/latest.html#arazzo-specification).

| Arzzo | Compatibility |
| ----- | ------------- |
| 1.0.0 | ✅            |

The default setting for this rule (in the `recommended` and `minimal` configuration) is `error`.

> This is an essential rule that should not be turned off except in rare and special cases.

Reviewer: ```suggestion
This is an essential rule. Do not turned it off, except in rare and special cases.
```
